modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1271,9 +1288,8 @@ private void _upsertNestedObjectEntries(
 						_getRelatedObjectDefinitionId(
 							objectDefinition, objectRelationship));
 
-				List<LinkedHashMap<String, Object>>
-					nestedObjectEntryPropertiesList =
-						(List<LinkedHashMap<String, Object>>)propertyValue;
+				List<Map<String, Object>> nestedObjectEntryPropertiesList =
```

Comment:
I think we should include, in some point, validations about the fields/relationships we are receiving in the parameter list. We could discuss it with Alex too next week